docs/source/htcondor.rst

```diff
@@ -0,0 +1,100 @@
+Using HTCondor with HTMap
+=========================
+
+Specification on exactly what "held", "idle", "running" and "completed" jobs
+mean is detailed at the `HTCondor documentation on machine states`_.
+
+.. _HTCondor documentation on machine states: https://htcondor.readthedocs.io/en/latest/admin-manual/policy-configuration.html#machine-states
+
+Requesting commonly used resources
+----------------------------------
+
+HTCondor's default configuration can be limiting -- what if your job requires
+more memory or more disk space? HTCondor can be configured to allow this, and
+HTMap supports the configuration via :class:`~htmap.MapOptions`.
+
+:class:`~htmap.MapOptions` accepts many of the same keys that `condor_submit`_
+accepts.  Some of the more commonly requested keys are:
+
+* ``request_memory``. Possible values are ``"1M`` for 1MB, ``"2GB"`` for 2GB of
+  memory. If not specified, the HTCondor's defaults are accepted provided
+  ``JOB_DEFAULT_REQUESTMEMORY`` is not set (one of the
+  `configuration variables`_).
+* ``request_cpus``. Possible values are ``"1"`` for 1 CPU.
+* ``request_disk``. Possible values are ``"10GB"`` for 10GB, ``"1T"`` for 1
+  terabytes.
+
+These would be set with :class:`~htmap.MapOptions`. For example, this code
+might be used:
+
+.. code:: python
+
+   options = ht.MapOptions(
+       request_cpus="1",
+       request_disk="10GB",
+       request_memory="4GB",
+   )
+   ht.map(..., map_options=options)
+
+When it's mentioned that "the option ``foo`` needs to be set" (possibly in a
+submit file), this corresponds to adding the option in the appropriate
+place in :class:`~MapOptions` (see the documentation for details).
```

Comment:
This link didn't work; I suspect this file is missing a `.. py:currentmodule:: htmap` at the top